modulefiles/build.container.intel.lua

```diff
@@ -0,0 +1,61 @@
+help([[
+Load environment to compile UFS_UTILS in a container using Intel
+]])
+
+hpss_ver=os.getenv("hpss_ver") or ""
+--load(pathJoin("hpss", hpss_ver))
```

Comment:
Why is hpss_ver here if its not loaded or used at buid time?

Reply:
I removed hpss and sfcio from the container modules.